client/state/db_bolt.go

```diff
@@ -138,6 +139,8 @@ var (
 
 	// nodeRegistrationKey is the key at which node registration data is stored.
 	nodeRegistrationKey = []byte("node_registration")
+
+	hostVolBucket = []byte("volumes_to_create")
```

Comment:
Just in case we decide to add CSI volumes here someday for some ungodly reason :grin: 

```suggestion
	hostVolBucket = []byte("host_volumes_to_create")
```